### PR TITLE
Milestone 4: Calculate.js and Operate.js

### PR DESCRIPTION
**Pull request by ldelbel**
Hey TSE,

in this milestone, I added calculate.js and operate.js with the suggested logic for receiving the data and processing the operations. I did it following my understanding of the instructions for the general functionality. Hopefully, it is correct.

I kindly request a review.

### CODE REVIEW

File: src/components/App.js
Comment (shloch) [inline src/components/App.js]: Here you were supposed to **ONLY** have `React.Fragment` (<>)  tag enclosing the two components, not a `<div>`. So line `6` and `11` wasn't necessary 
[[see milestone 2 requirement](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/ReactFragment.png)]

Comment (shloch) [reply src/components/App.js]: Coming back to `milestone4` you have to import `calculate.js` in this `<App>` component

File: src/logic/Calculate.js
Comment (shloch) [inline src/logic/Calculate.js]: this function is all lower-case spelled in requirement [[see requirement](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone4/onlyCalculate.png)]

this is not obligatory, it's just good practice so that we can easily have Component names in CamelCase, and we have no confusion between component names and function names

File: src/logic/Operate.js
Comment (shloch) [inline src/logic/Operate.js]: wise move

Comment (shloch) [reply src/logic/Operate.js]: anticipate division by Zero error for the division-operation (`5/0`). This will return an error in the next milestone. You might have well work on that scenario now.

**Review by shloch — CHANGES_REQUESTED** [review]
## Hello @ldelbel 

Great job with this milestone. All the parts and logic needed for this milestone are brought to life by you.
I however have multiple small fixes proposed for you to polish the whole thing.
I will therefore request you to apply these quick fixes prior to a final thumbs up.

Waiting for a final review.

HAPPY CODING

**Comment by ldelbel** [comment]
Hey TSE,

I had to add calculate() to the App.js so I could import calculate.js without ESLint errors.

**Review by shloch — APPROVED** [review]
GOOD TO GO
accomplished

HAPPY CODING

**Pull request merged**
